config/routes.rb

```diff
@@ -3,13 +3,14 @@
   get "/about", to: "pages#about"
   get "/sponsors", to: "pages#sponsors"
 
-  devise_for :admins, controllers: { invitations: "admins/invitations" }
+  devise_for :admins, controllers: { invitations: "admins/invitations", applications: "admins/applications" }
```

Comment:
You shouldn't add this in devise's routes since these are supposed to only work with devise (authentication only)